setup.py

```diff
@@ -72,14 +76,28 @@ def run(self):
     long_description = readme.read()
 
 
+def _get_public_version_string(scm_version):
+    return VERSION
+
+
+def _get_local_version_string(scm_version):
+    if 'dev' in VERSION:
+        return get_local_node_and_date(scm_version)
+    else:
+        return ''
```

Comment:
Can be done in a single line like `return get_local_node_and_date(scm_version) if 'dev' in VERSION else ''`